python/postprocessing/modules/btv/btagSFProducer.py

```diff
@@ -79,23 +90,50 @@ def __init__(self, era, algo = 'csvv2', sfFileName = None, verbose = 0):
                     },
                     'supported_wp' : [ "L", "M", "T", "shape_corr"]
                 },
-                '2017' : {
-                    'inputFileName' : "DeepCSV_94XSF_V4_B_F.csv",
+                '2016FastSim' : {
+                    'inputFileName' : "deepcsv_13TEV_16SL_18_3_2019.csv",
```

Comment:
Please add source to this file

Reply:
You are correct I did not add a twiki page for any of 2016 but here is the total twiki page so you can get to all other pages from here: https://twiki.cern.ch/twiki/bin/view/CMS/BtagRecommendation
Which is on this line:
https://github.com/susy2015/nanoAOD-tools/blob/Stop0l_dev5_Btag/python/postprocessing/modules/btv/btagSFProducer.py#L57
I will make that more clear. Also here is 2016 for all corrections: 
https://twiki.cern.ch/twiki/bin/viewauth/CMS/BtagRecommendation2016Legacy
I can also add that in if you would like.